src/main/java/net/mcreator/blockly/java/blocks/EventParameterSetBlock.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2025, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.blockly.java.blocks;
+
+import net.mcreator.blockly.BlocklyCompileNote;
+import net.mcreator.blockly.BlocklyToCode;
+import net.mcreator.blockly.IBlockGenerator;
+import net.mcreator.blockly.data.BlocklyLoader;
+import net.mcreator.blockly.data.ExternalTrigger;
+import net.mcreator.blockly.java.BlocklyToProcedure;
+import net.mcreator.generator.GeneratorWrapper;
+import net.mcreator.generator.template.TemplateGeneratorException;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.util.XMLUtil;
+import org.w3c.dom.Element;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class EventParameterSetBlock implements IBlockGenerator {
+
+	@Override public void generateBlock(BlocklyToCode master, Element block) throws TemplateGeneratorException {
+		if (master instanceof BlocklyToProcedure procedure) {

```

Comment:
Why is this check needed? I have not seen such check in any other blockly procedure block file

Reply:
@cdc12345, why was this ignored and not answered?